meetings/create/create_si_meeting.js

```diff
@@ -0,0 +1,129 @@
+/**
+ *                _
+ *  __      _____| |__   _____  __
+ *  \ \ /\ / / _ \ '_ \ / _ \ \/ /
+ *   \ V  V /  __/ |_) |  __/>  <         @WebexDevs
+ *    \_/\_/ \___|_.__/ \___/_/\_\
+ *
+ * CREATE a meeting in Webex with the REST API in Node
+ * https://developer.webex.com/docs/api/v1/meetings/create-a-meeting
+ *
+ * Step 0: Have a (free) Webex account: https://cart.webex.com/sign-up
+ * Step 1: Log in to https://developer.webex.com/login
+ * Step 2: Find your bearer token at
+ *         https://developer.webex.com/docs/getting-started under "Your
+ *         Personal Access Token" in the middle of the page.
+ * Step 3: Replace the string on the line that defines const myWebexDeveloperToken
+ *         and interpreterAry values for an interpreter object,
+ *         just below, with your personal bearer (access) token. Hit "save".
+ * Step 4: Run this file with node from within
+ *         this directory on the command line:
+ *
+ *         node ./create_si_meeting.js
+ *
+ * Step 5: Profit. Get your app listed in the Webex App Hub!
+ *         https://apphub.webex.com/
+ *
+ */
+
+ const https = require('https'); // https://nodejs.org/api/https.html
+
+ const myWebexDeveloperToken = 'REPLACE ME WITH YOUR WEBEX DEVELOPER PERSONAL ACCESS TOKEN';
+ //REPLACE languageCode1/2 VARIABLES WITH REQUIRED LANGUAGES, BELOW IS EXAMPLE OF ENGLISH TO SPANSISH AND SPANISH TO ENGLISH
```

Comment:
Are there links to docs we can add to this? Listing of supported language codes?